pkg/server/server.go

```diff
@@ -594,7 +594,13 @@ func (s *Server) Close() {
 func (s *Server) registerConn(conn *clientConn) bool {
 	s.rwlock.Lock()
 	defer s.rwlock.Unlock()
-	connections := len(s.clients)
+	// connections := len(s.clients)
+
+	connections := make(map[string]int, 0)
```

Comment:
What about maintaining a connectionID --> resourceGroup mapping to avoid traverse all connections every time.

Reply:
good idea

Reply:
fixed